demo/semantic-ui/index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>JQuery-validation demo | Semantic UI</title>
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/semantic-ui/2.1.4/semantic.min.css" />
+	<script type="text/javascript" src="../../lib/jquery-1.11.1.js"></script>
+	<script type="text/javascript" src="../../dist/jquery.validate.js"></script>
+	<style type="text/css">
+		.row {
+			padding-bottom: 0px !important;
+		}
+	</style>
+</head>
+<body>
+    <div class="ui grid sem">
+        <div class="four wide column"></div>
+
+        <div class="eight wide column">
+			<div class="ui info message" style="top: 15px;">
+				<div class="header">
+					This demo is using the Semantic UI framework
```

Comment:
Should this note changed to: `This demo is created using Semantic UI framework` ?